app/cfg/test0.json

```diff
@@ -0,0 +1,6 @@
+{
+    "I_MEM_OFFSET": 100,
```

Comment:
Let make configuration that make sense - such as a 4k I_mem+ 4k D_mem.
(with offsets starting from 0)

Reply:
will be done!